convert2rhel/checks.py

```diff
@@ -466,7 +467,13 @@ def check_package_updates():
     """Ensure that the system packages installed are up-to-date."""
     logger.task("Prepare: Checking if the packages are up-to-date")
 
-    packages_to_update = get_total_packages_to_update()
+    reposdir = get_hardcoded_repofiles_dir()
+
+    if (reposdir is None) and (not system_info.has_internet_access):
```

Comment:
This has a different meaning than suggested in https://github.com/oamg/convert2rhel/pull/458#discussion_r866281472.

The way you have it the check would be skipped only on offline systems we don't have hardcoded repofiles for, e.g. CentOS Linux 7, Alma Linux 8, etc.

However the purpose of the condition should be to skip the check in case we **have** the hardcoded repofiles for and the system is offline. The reason is that our hardcoded repofiles require an internet connection.

Reply:
Oh, my bad. I must have included the `reposdir is None` by mistake while trying to do something different. 

I will update it on Monday.

Reply:
done.